waspc/README.md

```diff
@@ -258,49 +258,56 @@ We can however still organize tests manually if we want in Tasty test trees, and
 Additionally, currently we limited tasty-discover to auto-detect only files ending with Test.hs (\*Test.hs glob).
 We might remove that requirement in the future if it proves to have no benefit.
 
-To summarize: If you are writing new tests, just put them in a file that ends with `Test.hs` in `test/` dir and that is it.
+To summarize: If you are writing new tests, just put them in a file that ends with `Test.hs` in `tests/` dir and that is it.
 
 For unit testing, we use **Hspec**.
 
 For property testing, we use **Quickcheck**.
 
 We additionally use **doctest** for testing code examples in documentation.
 
-All tests go into `test/` directory.
+All tests go into `tests/` directory.
 This is convention for Haskell, opposite to mixing them with source code as in Javascript for example.
 Not only that, but Haskell build tools don't have a good support for mixing them with source files, so even if we wanted to do that it is just not worth the hassle.
 
-Tests are run with `./run test`. This will run all the tests: unit, integration, and e2e.
+Tests are run with `./run test`. This will run all the tests (`waspc` + Wasp CLI + Wasp LS + `todoApp` + examples + starters).
```

Comment:
Hm, another place where we list the tests.

If we really want to list them in the readme, then we could do it once and point to that each time.

But I vote for not listing them at all